_includes/languageselection.html

```diff
@@ -45,10 +45,13 @@ <h5 class="modal-title" id="exampleModalLabel">{{texts[25]}}</h5>
                     Tiếng Việt</a>
                 <br>
                 <a href="#" data-dismiss="modal" ng-click="changeLang(12)"><span class="flag-icon flag-icon-cn"></span>
-                    中文</a>
+                    简体中文</a>
                 <br>
                 <a href="#" data-dismiss="modal" ng-click="changeLang(13)"><span class="flag-icon flag-icon-jp"></span>
                     日本語</a>
+                <br>
+                <a href="#" data-dismiss="modal" ng-click="changeLang(14)"><span class="flag-icon flag-icon-hk flag-icon-tw"></span>
+                    繁體中文</a>
```

Comment:
Can you correct the alphabetical order to include Japanese and Chinese?

Reply:
Done, please check again

Reply:
why two flags?
can you make it one?

Reply:
These both regions use Traditional Chinese as their official language, also I want add the Macau flag, but I not found it.

Reply:
![image](https://github.com/user-attachments/assets/58edc6ea-56e0-4afd-aebb-0ba5971dfce8)
Well two flags won't appear 🤷‍♂️
Delete `flag-icon-hk`